src/ipred.rs

```diff
@@ -791,7 +791,7 @@ fn get_filter_strength(wh: c_int, angle: c_int, is_sm: bool) -> c_int {
             (..=16, ..) => 0,
             (..=24, 4..) => 3,
             (..=24, ..) => 0,
-            (.., _) => 3,
+            (..) => 3,
```

Comment:
How does this work?

Reply:
I think this is `..` as in `{ .. }` rather than `..` as in `0..len`?